src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs

```diff
@@ -165,10 +162,66 @@ public async Task<IActionResult> SubmitClarification(Guid applicationId, RoatpFi
             {
                 return RedirectToAction(nameof(OpenApplications));
             }
+            var isClarificationFilesUpdate = HttpContext.Request.Form["submitClarificationFiles"].Count != 0;
+            var isClarificationOutcome = HttpContext.Request.Form["submitClarificationOutcome"].Count == 1;
+            vm.FinancialReviewDetails.ClarificationFiles = application.FinancialGrade.ClarificationFiles;
+            
+            if (isClarificationFilesUpdate)
+                vm.FilesToUpload = HttpContext.Request.Form.Files;
```

Comment:
Personally I would just assign this regardless as you're checking it below and also in the validator